src/core.ts

```diff
@@ -118,15 +118,15 @@ export const validatorCompiler: FastifySchemaCompiler<z.ZodTypeAny> =
     return { value: result.data };
   };
 
-const resolveSchema = (maybeSchema: z.ZodTypeAny | { properties: z.ZodTypeAny }) => {
+function resolveSchema(maybeSchema: z.ZodTypeAny | { properties: z.ZodTypeAny }): ZodSchema {
```

Comment:
Why is this not an arrow function? All others are using an arrow function.

Reply:
only the ones that are exported